.github/workflows/build.yml

```diff
@@ -131,6 +131,32 @@ jobs:
           nox -s test-py
         env:
           CARGO_TARGET_DIR: ${{ github.workspace }}/target
+      
+      - uses: dorny/paths-filter@v2
+        if: ${{ inputs.rust == 'stable' && inputs.python-version != 'pypy3.7' && inputs.python-version != 'pypy3.8' }}
+        id: ffi-changes
+        with:
+          filters: |
+            changed:
+              - 'pyo3-ffi/**'
+              - '.github/workflows/ci.yml'
+              - '.github/workflows/build.yml'
+      
+      - name: Checkout pyo3-ffi-check
+        # pypy 3.7 and 3.8 are not PEP 3123 compliant so fail checks here
+        if: ${{ steps.ffi-changes.outputs.changed == 'true' && inputs.rust == 'stable' && inputs.python-version != 'pypy3.7' && inputs.python-version != 'pypy3.8' }}
+        uses: actions/checkout@v3
+        with:
+          repository: PyO3/pyo3-ffi-check
```

Comment:
Should we import the code to `pyo3` repository instead? @davidhewitt

Reply:
Imported `pyo3-ffi-check` in https://github.com/PyO3/pyo3/pull/2771/commits/a14de763656735d2e623214e359727373088d583